lib/game.rb

```diff
@@ -1,2 +1,24 @@
 class Game
+  attr_reader :game_id, 
+              :season_id, 
+              :season_type, 
+              :game_date, 
+              :away_team_id, 
+              :home_team_id,
+              :away_goals,
+              :home_goals,
+              :venue_name,
+              :venue_link
+  def initialize(data)
+    @game_id = data[:game_id]
+    @season_id = data[:season]
+    @season_type = data[:type]
+    @game_date = data[:date_time]
+    @away_team_id = data[:away_team_id].to_i
```

Comment:
The conversion to integer here is good, we might want to do the same with all number values including the IDs.